source/data/globus_online_data_transfer.rst

```diff
@@ -102,7 +102,7 @@ partner clusters.
    +-----------+-------------------+--------------------+---------+---------------+
    | Cluster   | Display Name      | File Systems       | Site    | Access        |
    +===========+===================+====================+=========+===============+
-   | PPAN      | ncrc#dtn          | /lustre/f2/scratch | NCRC    | Trusted hosts |
+   | PPAN      | noaardhpcs#gaea   | /gpfs/f[56]        | NCRC    | Anywhere      |
```

Comment:
This is not the PPAN cluster which is accessible from noaardhpcs#gaea... it's Gaea...  (Guessing you want to update this, and create a PPAN entry?)